website/docs/quick-start-guide.md

```diff
@@ -495,18 +549,24 @@ insert into hudi_cow_pt_tbl partition(dt = '2021-12-09', hh='11') select 2, 'a2'
 ```
 
 **NOTICE**
-- By default,  if `preCombineKey `  is provided,  `insert into` use `upsert` as the type of write operation, otherwise use `insert`.
-- We support to use `bulk_insert` as the type of write operation, just need to set two configs: `hoodie.sql.bulk.insert.enable` and `hoodie.sql.insert.mode`. Example as follow: 
+- By default, records being inserted are treated as new insert records(from 0.14.0). You can configure different values for `hoodie.spark.sql.insert.into.operation` to change the behavior. 
+Possible values are "bulk_insert", "insert" and "upsert". If "bulk_insert" is chosen, hudi writes incoming records as is without any small file management.
+When "insert" (default) is chosen, hudi inserts the new incoming records and also does small file management. When "upsert" 
+is used, hudi takes upsert flow, where incoming batch will be de-duped before ingest and records will be upserted to ensure records get updated if a newer version is being 
+ingested. 
+- From 0.14.0, `hoodie.sql.bulk.insert.enable` and `hoodie.sql.insert.mode` are depecrated. Users are expected to use `hoodie.spark.sql.insert.into.operation` instead.
+- Where do we discuss the insert dup policy? fail, drop, none. I feel its too much of info in quick start.  
```

Comment:
Synced with @nsivabalan. We will tag this to the end of https://github.com/apache/hudi/pull/9622/files#diff-cb3da508fe853d0882b7057d91605d974a76bf55e9acc5508ccb813bcc015fdcR554  and link to appropriate config file.